docs/terms/index.mdx

```diff
@@ -0,0 +1,119 @@
+
+Module
+: A component or add-on for a Meshtastic device, typically referring to a hardware piece such as a LoRa radio module, that provides specific functionalities like communication or GPS tracking.
+
+MQTT
+: A lightweight messaging protocol designed for small sensors and mobile devices, enabling efficient data transmission in the Meshtastic network for Internet connectivity and integration with IoT platforms. See https://en.wikipedia.org/wiki/MQTT. In Meshtastic, MQTT is used to connect a node to the internet, and can be used to connect multiple meshes to each other.
+
+Node
+: A unit within the Meshtastic network that can send, receive, and relay messages, helping to form and extend the mesh network's coverage.
+
+NRF52
+: A chipset of microcontroller made by Nordic, used by a number of devices. Lower power usage than ESP32.
+
+Packet
+: A formatted unit of data sent over the network. In Meshtastic, packets carry messages, GPS locations, and other information through the LoRa mesh network.
+
+Protobuf
+: Protocol Buffers, a method developed by Google for serializing structured data, used in Meshtastic for efficient communication protocol between devices.
+
+PSK
+: Pre-Shared Key, a secret code or passphrase used in Meshtastic channels for encryption, ensuring that only devices with the matching PSK can communicate within that specific channel.
+
+Repeater
+: Infrastructure node for extending network coverage by relaying messages with minimal overhead. Not visible in Nodes list.
+
+Router
+: Infrastructure node for extending network coverage by relaying messages. Visible in Nodes list.
+
+rp2040
+: A microcontroller chip developed by Raspberry Pi, featuring dual ARM Cortex-M0+ processors.
+
+RX
+: Short for Receive.
+
+Sensor
+: A device component that detects and responds to some type of input from the physical environment. In Meshtastic, sensors can be used to gather environmental data (e.g., temperature, humidity, GPS location) which can then be transmitted over the mesh network for monitoring or other applications.
+
+Serial
+: A communication protocol used for the transmission of data between the Meshtastic device and a computer or other devices. Typically over USB or UART.
+
+SNR
+: Signal-to-Noise Ratio, a measure used in communications to quantify the level of a desired signal to the level of background noise. In Meshtastic and other wireless systems, a higher SNR indicates a clearer signal that can enhance the reliability and quality of data transmission.
+
+SWR
+: Standing Wave Ratio, a measure of the efficiency of the radio frequency (RF) power transmission from a transmitter through a transmission line to an antenna in Meshtastic devices. It indicates the ratio of the amplitude of a standing wave at maximum to the amplitude at minimum, with an ideal SWR close to 1:1, signifying that most of the power is transmitted to the antenna with minimal reflections.
+
+SX1262
```

Comment:
Not sure we need to define this.  If we do, we may want to define other radios.

Reply:
Yeah can't remember how that one was added and not the others. I don't think we need to define them here.